app/assets/javascripts/admin/controllers/admin-logs-staff-action-logs.js.es6

```diff
@@ -1,47 +1,75 @@
 import { exportEntity } from "discourse/lib/export-csv";
 import { outputExportResult } from "discourse/lib/export-result";
 import StaffActionLog from "admin/models/staff-action-log";
-import computed from "ember-addons/ember-computed-decorators";
+import {
+  default as computed,
+  observes,
+  on
+} from "ember-addons/ember-computed-decorators";
 
 export default Ember.Controller.extend({
   loading: false,
   filters: null,
   userHistoryActions: [],
+  model: [],
+  nextPage: 0,
+  lastPage: null,
 
   filtersExists: Ember.computed.gt("filterCount", 0),
+  showTable: Ember.computed.gt("model.length", 0),
 
-  filterActionIdChanged: function() {
-    const filterActionId = this.get("filterActionId");
-    if (filterActionId) {
+  @observes("filterActionId")
```

Comment:
Instead of using an observer here, is there a way you can trigger an event from the control when the value changes? It's always better to avoid observers if we can.